game-core/src/main/java/games/strategy/engine/framework/startup/ui/posted/game/pbem/EmailSenderEditor.java

```diff
@@ -181,8 +180,8 @@ private void testEmail() {
                 final int finalMessageType = messageType;
                 SwingUtilities.invokeLater(
                     () ->
-                        GameRunner.showMessageDialog(
-                            finalMessage, GameRunner.Title.of("Email Test"), finalMessageType));
+                        JOptionPane.showMessageDialog(
+                            null, finalMessage, "Email Test", finalMessageType));
```

Comment:
One difference this is probably going to cause is that the dialogs now no longer share the icon with the main frame, this problem can be found in different places too though.
Also it might be no longer on top of the mainframe

Reply:
The icon is significant, though the number of dialogs impacts is limited and they are not used that often. For now decided to favor dis-entangling things.